ui/lib/pki/addon/routes/roles/create.js

```diff
@@ -36,4 +36,10 @@ export default class PkiRolesCreateRoute extends Route {
       { label: 'create' },
     ];
   }
+
+  willTransition() {
```

Comment:
Resolves this test, multiple roles (with the same id) in the store
<img width="880" alt="Screenshot 2024-10-15 at 5 43 05 PM" src="https://github.com/user-attachments/assets/eff1f697-d5d2-4cdb-8211-9bf6e62e0ffa">